Formula/trojan-go.rb

```diff
@@ -0,0 +1,114 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go.git",
+      tag:      "v0.10.0",
+      revision: "a9ce6d4ada9143d58e8d1373d23f9ebfe871119e"
+  license "GPL-3.0-only"
+  head "https://github.com/p4gefau1t/trojan-go.git"
+
+  depends_on "go" => :build
+
+  resource "geoip" do
+    geoip_hash_url = "https://github.com/v2fly/geoip/releases/latest/download/geoip.dat.sha256sum"
+    url "https://github.com/v2fly/geoip/releases/latest/download/geoip.dat"
+    sha256 Utils.safe_popen_read("curl", "-L", geoip_hash_url)[0, 64]
```

Comment:
The `url` needs to be versioned and the `sha256` recorded in the formula. We don't want the build to change depending on the day it was built.

Reply:
Resolved.